src/core/TransformControls.tsx

```diff
@@ -56,7 +57,7 @@ export const TransformControls = React.forwardRef<TransformControlsImpl, Transfo
     const defaultCamera = useThree(({ camera }) => camera)
     const invalidate = useThree(({ invalidate }) => invalidate)
     const explCamera = camera || defaultCamera
-    const [controls] = React.useState(() => new TransformControlsImpl(explCamera, gl.domElement))
+    const [controls] = React.useState(() => new TransformControlsImpl(explCamera, domElement || gl.domElement))
```

Comment:
Should this be a `useMemo`? I know it wasn't the current PR decision to put it in `useState` but looking at it, what happens if someone changes the `camera` or `domElement`? I suppose the former is more likely... What do you think?

Reply:
@joshuaellis No idea, the `[state] = useState()` is pretty commonly used in Drei's hooks. I'm really not familiar with this syntax. I guess nothing will happen if we change the `camera` or `domElement`.
`const controls = useMemo(() => new TransformControlsImpl(explCamera, domElement || gl.domElement), [explCamera, domElement])` would seems more appropriate.

Reply:
It also feels wrong to not dispose anything in this component (`dispose={undefined}`). Or am I missing something?